csharp/extractor/Semmle.Autobuild/XmlBuildRule.cs

```diff
@@ -9,10 +9,16 @@ class XmlBuildRule : IBuildRule
     {
         public BuildScript Analyse(Autobuilder builder)
         {
-            var command = new CommandBuilder(builder.Actions).
+            var config = new CommandBuilder(builder.Actions).
                 RunCommand(builder.Odasa).
                 Argument("index --xml --extensions config");
-            return command.Script;
+            var csproj = new CommandBuilder(builder.Actions).
+                RunCommand(builder.Odasa).
+                Argument("index --xml --extensions csproj");
+            var props = new CommandBuilder(builder.Actions).
+                RunCommand(builder.Odasa).
+                Argument("index --xml --extensions props");
+            return config.Script & csproj.Script & props.Script;
```

Comment:
I think you can do it with just one command as `"index --xml --extensions config csproj props"`. Should we perhaps also index `.xml` files while we are at it?